docs/design/README.md

```diff
@@ -427,9 +427,9 @@ and [`while`](#while), and
 
 ### Integer types
 
-The signed-integer type with bit width `N` may be written `iN` or
-`Carbon.Int(N)`, as long as `N` is a positive multiple of 8. For example, `i32`
-is equivalent to `Carbon.Int(32)`. Signed-integer
+The signed-integer type with bit width `N` may be written `iN`, as long as `N`
+is a positive multiple of 8. For example, `i32` is equivalent to
+`Carbon.Int(32)`. Signed-integer
```

Comment:
Should this say `Core.Int(32)`?

Reply:
I rewrote to not mention the `Core.Int` type. That is a detail that I think is fine being only in `expressions/literals.md`.